dvc/utils/__init__.py

```diff
@@ -420,3 +420,8 @@ def resolve_output(inp, out):
     if os.path.isdir(out):
         return os.path.join(out, name)
     return out
+
+
+def is_exec_found(exec_name):
+    cmd = "({}) 2>/dev/null".format(exec_name)
+    return hasattr(os, "system") and os.system(cmd) == 0
```

Comment:
I'm not aware of any OS that we support that doesn't have `system` implemented. Was it just inspired by pydoc's implementation or is there a good reason to do this?

Reply:
i'm confused by this, too.
this is a copy/paste from the `pydoc`. should i make it?
```
return os.system(cmd) == 0
```

Reply:
@xliiv In that case, I would just inline it into the `if` directly, no need for a separate function (also `is_exec_found` is a very confusing name 🙂).

Reply:
agree

Reply:
thanks and done.
should i resolved this comment?

Reply:
https://github.com/iterative/dvc/pull/2815#discussion_r354017153